TouchPortalAPI/TpToPy.py

```diff
@@ -0,0 +1,140 @@
+import json
+from pathlib import Path
+
+class TpToPy():
+    def __init__(self, entry):
+        self.entry = json.loads(Path(entry).resolve().read_text())
+    
+    def getPluginId(self):
+        return self.entry.get("id", "")
+
+    def __convertData(self, data):
+        newData = {}
+        if isinstance(data, list):
+            for item in range(len(data)):
```

Comment:
This.... increments the index value on each key by +1 ?

Reply:
Ah this part? `newData[item+1] = data[item]` yeah I can remove this `+1` since it's not going to be shown in doc.

Reply:
So I wold probably call this __arrayToDict() to make it clearer :-)   Also... it's replacing a one-liner... `newData  = dict(zip(i := iter(data), i))`  But no matter.